s4e-backend/src/main/java/pl/cyfronet/s4e/data/repository/ProductCategoryRepository.java

```diff
@@ -0,0 +1,16 @@
+package pl.cyfronet.s4e.data.repository;
+
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.transaction.annotation.Transactional;
+import pl.cyfronet.s4e.bean.ProductCategory;
+
+public interface ProductCategoryRepository extends CrudRepository<ProductCategory, Long> {
```

Comment:
I'd add a `@Transactional(readOnly=true)` on the repository, the same as it is in other repos.

Reply:
I wouldn't do that, because
1. We need to seed using this repository (needs write rights)
2. We need to mock categories in tests (needs write rights)